doc/source/conf.py

```diff
@@ -83,11 +83,9 @@
         
     .. nbinfo::
         Interactive online version:
-        :raw-html:`<a href="https://mybinder.org/v2/gh/liam2/larray/{{ env.config.release }}?
+        :raw-html:`<a href="https://mybinder.org/v2/gh/alixdamman/larray/{{ env.config.release }}?
```

Comment:
?

Reply:
read the commit message

Reply:
Ok, clear now. 

PS: I usually do not read individual commits, unless you somehow hint at it in the PR notes... which I just noticed you actually **did**. I just somehow missed that comment. Sorry about this.